src/main/java/juja/microservices/gamification/controller/AchievementController.java

```diff
@@ -26,6 +30,15 @@
 
     @RequestMapping(value = "/achieve/daily", method = RequestMethod.POST)
     @ResponseBody
+    @ApiOperation(
+            value = "Add points for daily report",
+            notes = "This method adds points for daily report"
+    )
+    //TODO @ApiImplicitParams doesn`t works
+    //    @ApiImplicitParams(value - @ApiImplicitParam(name, value, dataType, required))
+    @ApiResponses(value = {
+            @ApiResponse(code = HttpURLConnection.HTTP_OK, message = "Returns achievement id.")
+    })
```

Comment:
You can add in ApiResponses another ApiResponse with error codes and parameter 'response = ApiErrorMessage.class'